### ISSUE TITLE
Error when running npm run dev

### ISSUE THREAD

**Issue by supa-freak** [carstenlebek/shopify-node-app-starter#10]
Hi!
After following all the steps, when running npm run dev, the software runs into an error. Output is:
```
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from [my-path]\.env
Abort [Error]: The CLIKitStore instance hasn't been initialized
    at cliKitStore (file:///[MY-APP-PATH]/node_modules/@shopify/cli-kit/dist/store.js:28:15)
    ......
```

Ani ideas why that is?

**Comment by alexandrosk** [comment]
`npm install --force? ` 

Did you install it like this? 

seems like shopify cli is missing

**Comment by supa-freak** [comment]
Yes, did, otherwise it doesnt install at all.

**Comment by carstenlebek** [comment]
Hey @supa-freak 

I have found the issue and fixed it in 6e5048ba41c1c4d61d6a5d7f57da6b7134aa134d.

It seems like the cliKit store is initialized globaly, so if you haven't used the Shopify CLI before, you get the error.
